java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -105,12 +105,14 @@ public String toString() {
     public final ProgramState programState;
     @Nullable
     public Node parent;
+    public final List<Node> cacheHits;
```

Comment:
This is poorly named : cache hit has absolutely no significance here... Those are parents nodes.